source/scpp/src/scp/SCPDriver.cpp

```diff
@@ -98,4 +98,37 @@ SCPDriver::computeTimeout(uint32 roundNumber)
     }
     return std::chrono::seconds(timeoutInSeconds);
 }
+
+Value SCPDriver::extractValidValue(uint64 slotIndex, Value const& value)
```

Comment:
I'm pretty sure you could revert this change if you remove `/Zc:inline`.
See the doc https://docs.microsoft.com/en-us/cpp/build/reference/zc-inline-remove-unreferenced-comdat?view=vs-2019